app/assets/stylesheets/application.css

```diff
@@ -34,3 +34,9 @@ input[type="submit"] {
     color: blue;
     cursor: pointer;
   }
+
+  form {
+       display:inline;
+       margin:0px;
+       padding:0px;
```

Comment:
I added this CSS so that the delete button and the update button were on the same line